### ISSUE TITLE
QA: Do not force user-input on wine-{mono,gecko} for new wineprefix

### ISSUE THREAD

**Issue by Kreyren** [PhoenicisOrg/phoenicis#1968]
While i try to install new application on phoenicis it usually prompts me with 
![image](https://user-images.githubusercontent.com/11302521/58975685-302d7480-878b-11e9-8bde-7277ad83b9c9.png)

Which is annoying.. -> Do not prompt for user-input

**Comment by plata** [comment]
It's not like we actively open this window... I don't know a way to prevent this, so unless you can prove that it's possible I will close this issue as "invalid".

**Comment by qparis** [comment]
The only way to prevent it is to provide a gecko/mono runtime with Wine

**Comment by plata** [comment]
Do we want to do that? If not, we should close the issue.

**Comment by qparis** [comment]
Yes we absolutely do

**Comment by plata** [comment]
So we move this to winebuild?

**Comment by qparis** [comment]
This is not really related to winebuild. This is more transveral feature. Winebuild is really few impacted in fact

**Comment by plata** [comment]
Ah ok so not the part of that Wine runtime that we have currently.

**Comment by qparis** [comment]
We have three components that are impacted: 
- scripts : gecko and mono are pretty much managed like the runtime 
- webservice : this needs to be released open source or rewritten as a micro service. At the moment, it is a small PHP code that runs on the website 
- winebuild? : A component that finds what is the right gecko archive for a given wine version

**Comment by plata** [comment]
Understood.

**Comment by Kreyren** [comment]
There is also silent way to install mono.. https://gitweb.gentoo.org/repo/gentoo.git/tree/app-emulation/wine-mono/wine-mono-4.8.3.ebuild

same for gecko afaik

**Comment by qparis** [comment]
This is what we are talking about

**Comment by ImperatorS79** [comment]
The only way to know the right gecko/mono version for a certain wine version is to look at https://wiki.winehq.org/Gecko and https://wiki.winehq.org/Mono. However, for all wine > 2.0, you can always use latest gecko and mono it seems.

**Comment by Kreyren** [comment]
Or we can probably solve this by providing verb assuming that there might be wineapps that breaks if mono/gecko is present.

**Comment by qparis** [comment]
Nope. A verb works inside a prefix and once the prefix is created, it is already too late, the window has already been shown

**Comment by Kreyren** [comment]
@qparis install it prior to making wineprefix then

**Comment by qparis** [comment]
Then it is not a verb and it is installed aside the wineversion, exactly what we are talking about
